pkg/apis/serving/v1/revision_validation.go

```diff
@@ -188,14 +188,15 @@ func validateQueueSidecarAnnotation(m map[string]string) *apis.FieldError {
 	if !ok {
 		return nil
 	}
+	errs := apis.ErrGeneric("Queue proxy resource percentage annotation is deprecated. Please use the available annotations to explicitly set resource values per service").ViaKey(k).At(apis.WarningLevel)
```

Comment:
We should validate the new annotations

Reply:
@dprotaso  I just followed the code for "resourcePercentage" validation that happens when we create the qp container. I didn't want to change the UX. Should I now?

Reply:
Ok will add it.